### ISSUE TITLE
Update GFU to use new I3RateDb tables in I3Live (Trac #323)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#322]
Migrated from https://code.icecube.wisc.edu/ticket/323
```json
{
    "status": "closed", 
    "changetime": "2019-01-11T20:04:01", 
    "description": "", 
    "reporter": "blaufuss", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1547237041230692", 
    "component": "jeb + pnf", 
    "summary": "Update GFU to use new I3RateDb tables in I3Live", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2011-11-03T15:02:40", 
    "milestone": "", 
    "owner": "tschmidt", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150211T19:46:16: 
* blaufuss changed status from "new" to "closed"
* blaufuss changed resolution from "" to "wontfix"

**Comment by icecube-trac** [comment]
Trac update at 20190111T20:04:01: 
* nwhitehorn changed resolution from "wontfix" to "fixed"
* nwhitehorn commented:

> In changeset 168394:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="168394"
> add outbox to docs fixes #322
> ```
